__tests__/app.test.js

```diff
@@ -56,9 +56,49 @@ describe('GET /api', () => {
             });
     });
 
-    test('should return 404 error if wrong input', () => {
-        return request(app)
-            .get('/apy')
-            .expect(404);
-    });
+   
 });
+
+describe('GET /api/articles/:article_id', () => {
+    test('should respond with 200 status code for good request', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            const keys = Object.keys(body.articles)
+            if (keys.length !== 0) {
+                expect(body.articles).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        body: expect.any(String), 
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        article_img_url: expect.any(String)
+
+
```

Comment:
Suggestion: You can test for most of the actual object here since you know the id